apps/coscrad-frontend/src/components/resources/videos/video-detail.thumbnail.presenter.tsx

```diff
@@ -1,19 +1,26 @@
-import { ICategorizableDetailQueryResult, IVideoViewModel } from '@coscrad/api-interfaces';
+import {
+    ICategorizableDetailQueryResult,
+    IVideoViewModel,
+    ResourceType,
+} from '@coscrad/api-interfaces';
 import { SinglePropertyPresenter } from '../../../utils/generic-components';
 import { ResourceDetailThumbnailPresenter } from '../../../utils/generic-components/presenters/detail-views';
 import { convertMillisecondsToSeconds } from '../utils/math';
 
 export const VideoDetailThumbnailPresenter = ({
+    id,
     lengthMilliseconds,
     text: plainText,
     name,
-}: ICategorizableDetailQueryResult<IVideoViewModel>): JSX.Element => (
-    <ResourceDetailThumbnailPresenter name={name}>
-        <SinglePropertyPresenter
-            display="Duration"
-            value={convertMillisecondsToSeconds(lengthMilliseconds)}
-        />
-        <h3>Transcript:</h3>
-        <p>{plainText}</p>
-    </ResourceDetailThumbnailPresenter>
-);
+}: ICategorizableDetailQueryResult<IVideoViewModel>): JSX.Element => {
+    return (
+        <ResourceDetailThumbnailPresenter id={id} name={name} type={ResourceType.video}>
+            <SinglePropertyPresenter
+                display="Duration"
+                value={convertMillisecondsToSeconds(lengthMilliseconds)}
+            />
+            <h3>Transcript:</h3>
+            <p>{plainText}</p>
```

Comment:
Ideally, we should have a transcript presenter so we have a SST for audio and video transcript presentation. We could refactor when we do audio item's presenters.

Reply:
Sounds good.  Do we need a story for this?

Reply:
Added a story: https://www.pivotaltracker.com/story/show/184666073